src/validators/mod.rs

```diff
@@ -576,3 +576,13 @@ pub trait Validator: Send + Sync + Clone + Debug {
         Ok(())
     }
 }
+
+#[pyclass(module = "pydantic_core._pydantic_core")]
+pub struct ValidatorInfo {
+    #[pyo3(get)]
+    data: Option<Py<PyDict>>,
+    #[pyo3(get)]
+    config: PyObject,
+    #[pyo3(get)]
+    context: Option<Py<PyAny>>,
```

Comment:
`PyObject` is an alias of `Py<PyAny>`

Reply:
Yep thanks will replace to make it the same everywhere